build.sh

```diff
@@ -2,6 +2,12 @@
 set -eu -o pipefail
 exec 1>&2 <&-
 
+declare -r -a GO_PACKAGES_ALL=(
```

Comment:
There are also some go tests in 
https://github.com/SAP/stewardci-core/tree/add-vet-to-build-script/charts/steward/test 

May be they are special because they don't test go code but helm generation but they should not be dropped.

Reply:
Fixed by c554461.